lib/rubocop/cop/style/map_into_array.rb

```diff
@@ -53,12 +53,17 @@ class MapIntoArray < Base
 
         MSG = 'Use `%<new_method_name>s` instead of `each` to map elements into an array.'
 
+        # @!method suitable_element?(node)
+        def_node_matcher :suitable_element?, <<-PATTERN
```

Comment:
How about a name like this?

```suggestion
        def_node_matcher :suitable_argument_node?, <<-PATTERN
```